src/main/java/seedu/eylah/expensesplitter/logic/commands/PaidCommand.java

```diff
@@ -12,24 +16,47 @@
 
     public static final String COMMAND_WORD = "paid";
 
-    public static final String MESSAGE_SUCCESS = "Successfully deducted amount from Person named : ";
+    public static final String MESSAGE_SUCCESS = "Successfully deducted amount from Person named: ";
 
 
-    private final Person personPaid;
+    //private final Person personPaid;
+    private final int indexOfPersonPaid;
+    private final String amountPaid;
 
     /**
      * Creates an PaidCommand to add the specified {@code Person}
      */
-    public PaidCommand (Person person) {
-        requireNonNull(person);
-        personPaid = person;
+    public PaidCommand (int indexOfPersonPaid, String amountPaid) {
```

Comment:
Actually we have an `Index` class in the common->core->index, maybe you can consider using it instead of pure integer index. The `Index` class provides `getOneBased()` and `getZeroBased()` method which can make your index easier to trace.

Reply:
alright understood! Thank you